### PR TITLE
docs: Reorganize docs to create less tabs across the top

### PR DESCRIPTION
**Pull request by ejlee3**
## PR Checklist

- [x] Added label to the Pull Request for easier discoverability and search
- [x] Commit Message meets guidelines as indicated in the URL https://github.com/intel-retail/automated-self-checkout/blob/main/CONTRIBUTING.md
- [x] Every commit is a single defect fix and does not mix feature addition or changes
- [ ] Unit Tests have been added for new changes
- [x] Updated Documentation as relevant to the changes
- [ ] All commented code has been removed
- [ ] If you've added a dependency, you've ensured license is compatible with repository license and clearly outlined the added dependency.
- [ ] PR change contains code related to security
- [ ] PR introduces changes that breaks compatibility with other modules (If YES, please provide details below)

## Issue this PR will close

close: #**issue_number**

## Test Instructions if applicable
build docs using `make serve-docs`

### CODE REVIEW

**Review by brian-intel — APPROVED** [review]

**Review by antoniomtz — APPROVED** [review]

File: mkdocs.yml [hunk 1]
```diff
@@ -24,32 +24,37 @@ repo_name: intel-retail/automated-self-checkout
...
+        - Quick Start Run Pipeline: './OVMS/quick_pipelinerun.md'
+        - Advanced Features:
+          - Custom Run: './OVMS/pipelinerun.md'
+          - Object Detection Pipeline: './OVMS/runObjectDetectionPipelineWithNewModel.md'
+          - C-API Face Detection Pipeline: './OVMS/capiPipelineRun.md'
+          - Supporting Different Languages: './OVMS/supportingDifferentLanguage.md'
+          - Supporting Different Models: './OVMS/supportingDifferentModel.md'
+          - Run Pipelines with Docker-Compose for Developer Toolbox: './OVMS/pipelineDockerCompose.md'
+      - Run Benchmarking:
+        - Quick Start Benchmarking: 'pipelinebenchmarking.md'
```
Comment (jim-wang-intel) [inline mkdocs.yml:41]:
```suggestion
        - Quick Start Benchmarking: 'pipelinebenchmarking.md'  (WIP)?
```

Comment (ejlee3) [reply mkdocs.yml:41]: I don't think we can add WIP there in the mkdocs file

**Review by vli11 — APPROVED** [review]
lgtm

**Review by jim-wang-intel — APPROVED** [review]
LGTM

**Pull request merged**